src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -431,6 +440,43 @@ export default class DataTable extends Vue {
     }
   }
 
+  _radioButton(headRow: boolean, rowLevel: string = '', rowData: DataTableRow | null = null) {
```

Comment:
`rowLevel` is one of these: `'parent' | 'child' | 'grandChild'`
I suggest defining a type for this and for row() method.